pkg/cloud/services/compute/service.go

```diff
@@ -29,11 +29,8 @@ import (
 )
 
 type Service struct {
-	provider          *gophercloud.ProviderClient
 	projectID         string
-	computeClient     *gophercloud.ServiceClient
-	identityClient    *gophercloud.ServiceClient
-	imagesClient      *gophercloud.ServiceClient
+	computeService    Client
```

Comment:
I guess it's a philosophical difference at what point a Client becomes a Service. :smile: I think this is fine, the networking service is definitely a wrapper around the networking client. If the bare compute client serves its purpose, it makes sense to have `computeService` and `networkingService` named consistently in this struct.

Reply:
Yeah, it's a client. This is easy to fix so I'll update.